setup.cfg

```diff
@@ -0,0 +1,25 @@
+[metadata]
+name = config-probe
+url = https://github.com/internap/python-config-probe
+author = Internap Hosting
+author-email = opensource@internap.com
+summary = Auto-discovery of configurations for easy inline use
+description-file =
+    README.md
+classifier =
```

Comment:
Don't know if this is necessary for the pypi info, but you *could* add:

license = Apache-2

We have this in a couple of our existing projects too.